snowpack/src/commands/dev.ts

```diff
@@ -368,18 +368,6 @@ export async function startServer(
     }
   }
 
-  function loadUrl(
-    reqUrl: string,
-    opt?: (LoadUrlOptions & {encoding?: undefined}) | undefined,
-  ): Promise<LoadResult<Buffer | string>>;
-  function loadUrl(
-    reqUrl: string,
-    opt: LoadUrlOptions & {encoding: BufferEncoding},
-  ): Promise<LoadResult<string>>;
-  function loadUrl(
-    reqUrl: string,
-    opt: LoadUrlOptions & {encoding: null},
-  ): Promise<LoadResult<Buffer>>;
```

Comment:
I only deleted wanted to raise the question: why is this needed? I’m confused by these typings

Reply:
if it compiles without this, then you're good to remove! The variations are already kept in `types.ts` which is what the user will see.

This makes it clear what the return value is, depending on whether you pass something like `encoding: 'utf8'` or not.